src/core/libraries/videoout/driver.cpp

```diff
@@ -269,8 +269,8 @@ void VideoOutDriver::SubmitFlipInternal(VideoOutPort* port, s32 index, s64 flip_
 }
 
 void VideoOutDriver::PresentThread(std::stop_token token) {
-    static constexpr std::chrono::nanoseconds VblankPeriod{16666667};
-    const auto vblank_period = VblankPeriod / Config::vblankDiv();
+    static constexpr std::chrono::nanoseconds VblankPeriod{1000000000};
+    const auto vblank_period = VblankPeriod / Config::vblankFreq();
```

Comment:
```suggestion
    const std::chrono::nanoseconds vblank_period(1000000000 / Config::vblankFreq());
```

Since `VblankPeriod` is no longer actually a vblank period, just the number of nanoseconds in a second.